tests/containers/nodejs/Dockerfile

```diff
@@ -9,5 +11,6 @@ RUN mkdir /app
 ADD fibonacci.js /app
 
 ENV NODE_RUNTIME_FLAGS ${NODE_RUNTIME_FLAGS}
+USER 1001
```

Comment:
Why this change?

Reply:
I wasn't able to create /app directory on centos image. Can we f.e. switch dir to /tmp/app instead of /app?

Reply:
Yes we can, no problem

Reply:
We can leave this here as well

Reply:
Or was it needed for hadolint and that's it?

Reply:
Actually I think that linter doesn't scan this file, I removed this line